samples/README.md

```diff
@@ -1,45 +1,170 @@
 [//]: # "This README.md file is auto-generated, all changes to this file will be lost."
-[//]: # "To regenerate it, use `npm run generate-scaffolding`."
+[//]: # "To regenerate it, use `python -m synthtool`."
 <img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google Cloud Platform logo" title="Google Cloud Platform" align="right" height="96" width="96"/>
 
-# Google Compute Engine: Node.js Samples
+# [Google Compute Engine: Node.js Samples](https://github.com/googleapis/nodejs-compute)
 
 [![Open in Cloud Shell][shell_img]][shell_link]
 
-[Compute Engine](https://cloud.google.com/compute/docs/) lets you create and run virtual machines on Google infrastructure. Compute Engine offers scale, performance, and value that allows you to easily launch large compute clusters on Google&#x27;s infrastructure. There are no upfront investments and you can run thousands of virtual CPUs on a system that has been designed to be fast, and to offer strong consistency of performance.
+
 
 ## Table of Contents
 
 * [Before you begin](#before-you-begin)
 * [Samples](#samples)
-  * [Create Virtual Machine with Apache and Custom Homepage](#create-virtual-machine-with-apache-and-custom-homepage)
-  * [List Virtual Machines](#list-virtual-machines)
+  * [Create VM](#create-vm)
+  * [Delete VM](#delete-vm)
+  * [List V Ms](#list-v-ms)
```

Comment:
likewise, we providing some metadata should fix this weirdness.

Reply:
Fixed :)